infra/modules/aws-bulk/main.tf

```diff
@@ -9,20 +9,23 @@ terraform {
 }
 
 module "psoxy-aws" {
-  source = "../aws"
+  source = "git::https://github.com/worklytics/psoxy//infra/modules/aws?ref=v0.3.0-beta.3"
```

Comment:
this is inside a module; modules should reference each other locally I think, or things get really weird.

Reply:
of course, imho it should not be inside a module; it should be moved up to a configuration, as this needs to be done once per AWS deployment (not for every lambda)